app/models/external_services_redis/status.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'common/models/redis_store'
+require 'common/models/concerns/cache_aside'
```

Comment:
requires not needed because we are using autoload/eagerload

Reply:
@lihanli Nice.  Can you pls link me to that?

Reply:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/application.rb#L45-L47

Reply:
Thanks!